hypha/apply/projects/templates/application_projects/includes/supporting_documents.html

```diff
@@ -4,7 +4,12 @@
 
 <div class="docs-block wrapper--outer-space-large" {% if collapsible_header %} x-data="{ collapsed: false }" {% endif %}>
     <div class="docs-block__header" {% if collapsible_header %} x-on:click="collapsed = ! collapsed" {% endif %}>
```

Comment:
As this is non-standard actionable html element please add aria-label and `role=button` to it.

Reply:
see this for more details: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/

Reply:
I believe it should be possible here.

Reply:
As it is a conditional accordion so `details` and `summary` tag doesn't make much sense here(it acts as an accordion only after a stage). But yes for the accessibility I agree I would add aria-label and aria-expanded and role to it.